po/pt_BR.po

```diff
@@ -282,123 +282,123 @@ msgstr ""
-msgstr ""
+msgstr "Com certeza!"
 
 #: data/metainfo.appdata.xml.in:200
 msgid "Initial release"
-msgstr ""
+msgstr "Release inicial"
 
 #: src/FortuneLabel.vala:16
 msgid "It is certain."
-msgstr ""
+msgstr "Isso é certo."
 
 #: src/FortuneLabel.vala:24
 msgid "Without a doubt."
-msgstr ""
+msgstr "Sem dúvida."
 
 #: src/FortuneLabel.vala:28
 msgid "Yes—definitely."
-msgstr ""
+msgstr "Definitivamente sim."
 
 #: src/FortuneLabel.vala:32
 msgid "You may rely on it."
-msgstr ""
+msgstr "Pode confiar."
 
 #: src/FortuneLabel.vala:36
 msgid "As I see it, yes."
-msgstr ""
+msgstr "Da forma que eu vejo, sim."
 
 #: src/FortuneLabel.vala:40
 msgid "Most likely."
-msgstr ""
+msgstr "Muito provavelmente."
 
 #: src/FortuneLabel.vala:44
 msgid "Outlook good."
-msgstr ""
+msgstr "Parece bom."
 
 #: src/FortuneLabel.vala:48
 msgid "Yes."
-msgstr ""
+msgstr "Sim."
 
 #: src/FortuneLabel.vala:52
 msgid "Signs point to yes."
-msgstr ""
+msgstr "Sinais me apontam que sim."
 
 #: src/FortuneLabel.vala:56
 msgid "Yep!"
 msgstr ""
 
 #: src/FortuneLabel.vala:60
 msgid "Absolutely!"
-msgstr ""
+msgstr "Absolutamente1"
 
 #: src/FortuneLabel.vala:64
 msgid "You bet!"
-msgstr ""
+msgstr "Pode apostar!"
 
 #: src/FortuneLabel.vala:68
 msgid "Confirmed."
-msgstr ""
+msgstr "Confirmado."
 
 #: src/FortuneLabel.vala:72
 msgid "Don’t count on it."
-msgstr ""
+msgstr "Não conte com isso."
 
 #: src/FortuneLabel.vala:76
 msgid "My reply is no."
-msgstr ""
+msgstr "Minha resposta é não."
 
 #: src/FortuneLabel.vala:80
 msgid "My sources say no."
-msgstr ""
+msgstr "Minhas fontes dizem que não."
 
 #: src/FortuneLabel.vala:84
 msgid "Outlook not so good."
-msgstr ""
+msgstr "A perspectiva não é boa."
 
 #: src/FortuneLabel.vala:88
 msgid "Very doubtful."
-msgstr ""
+msgstr "Bastante duvidoso."
 
 #: src/FortuneLabel.vala:92
 msgid "Naw."
-msgstr ""
+msgstr "Não."
 
 #: src/FortuneLabel.vala:96
 msgid "I’ve got a bad feeling about this…"
-msgstr ""
+msgstr "Eu tenho um mau pressentimento sobre isso..."
 
 #: src/FortuneLabel.vala:100
 msgid "Reply hazy, try again."
-msgstr ""
+msgstr "Resposta nebulosa, tente novamente."
 
 #: src/FortuneLabel.vala:104
 msgid "Ask again later."
-msgstr ""
+msgstr "Pergunte novamente em outra hora."
 
 #: src/FortuneLabel.vala:108
 msgid "Better not tell you now."
-msgstr ""
+msgstr "Melhor não te contar agora."
 
 #: src/FortuneLabel.vala:112
 msgid "Cannot predict now."
-msgstr ""
+msgstr "Não é possível prever agora."
 
 #: src/FortuneLabel.vala:116
 msgid "Concentrate and ask again."
-msgstr ""
+msgstr "Se concentre e pergunte novamente."
 
 #: src/FortuneLabel.vala:120
 msgid "Impossible to see, the future is."
-msgstr ""
+msgstr "Impossível ver, o futuro é."
 
 #: src/FortuneLabel.vala:124
 msgid "404 Answer Not Found"
-msgstr ""
+msgstr "Erro 404 Resposta não encontrada"
 
 #: src/MainWindow.vala:18
 msgid "About"
-msgstr ""
+msgstr "Sobre"
 
 #. / The translator credits. Please translate this with your name(s).
```

Comment:
If you'd like credit in the UI for the translation, could you add your name to the translator-credits?